canonical/snapcraft.io
# Removed legacy settings page

## Done
Removed the legacy code for the snap settings page

## QA
- Go to https://snapcraft-io-4089.demos.haus/SNAP_NAME/settings
- Check that the form works as expected
